Backpack/src/main/res/values/backpack.text.xml

```diff
@@ -20,63 +20,26 @@
 
   <!-- THIS FILE IS AUTO GENERATED. DO NOT MODIFY! -->
 
+
   <declare-styleable name="BpkTextStyle">
     <attr name="android:fontFamily" />
     <attr name="android:textSize" />
     <attr name="android:letterSpacing" />
     <attr name="lineHeight" />
   </declare-styleable>
-
-  <dimen name="bpkTextXlSize">24sp</dimen>
-  <dimen name="bpkTextXxxlSize">40sp</dimen>
-  <dimen name="bpkTextSmSize">14sp</dimen>
-  <dimen name="bpkTextXsSize">12sp</dimen>
-  <dimen name="bpkTextLgSize">20sp</dimen>
-  <dimen name="bpkText5XlSize">64sp</dimen>
-  <dimen name="bpkText6XlSize">76sp</dimen>
-  <dimen name="bpkText7XlSize">96sp</dimen>
-  <dimen name="bpkText8XlSize">120sp</dimen>
-  <dimen name="bpkTextXxlSize">32sp</dimen>
-  <dimen name="bpkTextXxxxlSize">48sp</dimen>
-  <dimen name="bpkTextBaseSize">16sp</dimen>
-
-  <style name="bpkTextHero1">
-    <item name="android:fontFamily">?bpkFontFamilyEmphasized</item>
-    <item name="android:textColor">@color/bpkTextPrimary</item>
-    <item name="android:textSize">@dimen/bpkText8XlSize</item>
-    <item name="lineHeight">120sp</item>
-    <item name="android:letterSpacing">-0.02</item>
-  </style>
-
-  <style name="bpkTextHero2">
-    <item name="android:fontFamily">?bpkFontFamilyEmphasized</item>
-    <item name="android:textColor">@color/bpkTextPrimary</item>
-    <item name="android:textSize">@dimen/bpkText7XlSize</item>
-    <item name="lineHeight">96sp</item>
-    <item name="android:letterSpacing">-0.02</item>
-  </style>
-
-  <style name="bpkTextHero3">
-    <item name="android:fontFamily">?bpkFontFamilyEmphasized</item>
-    <item name="android:textColor">@color/bpkTextPrimary</item>
-    <item name="android:textSize">@dimen/bpkText6XlSize</item>
-    <item name="lineHeight">84sp</item>
-    <item name="android:letterSpacing">-0.02</item>
-  </style>
-
-  <style name="bpkTextFootnote">
+  
+  <style name="bpkTextBodyDefault">
```

Comment:
the order is different, but they're the same. the screenshot tests will validate that